.vagrant/rgloader/loader.rb

```diff
@@ -0,0 +1,9 @@
+# This file loads the proper rgloader/loader.rb file that comes packaged
```

Comment:
If files are already being tracked by git, adding them to .gitignore is not enough. You can run e.g. `git rm --cached .vagrant/rgloader/loader.rb` to "delete" the file from the repository while keeping it locally.

Reply:
Deleted as advised